controllers/tasks/replication/demote.go

```diff
@@ -0,0 +1,15 @@
+package replication
+
+type DemoteImageTask struct {
```

Comment:
it should be demoteVolume. Need to replace Image with Volume in other places also